resources/admin.rb

```diff
@@ -19,13 +19,15 @@
       client.create_cluster_admin(username, password)
       updated_by_last_action true
     end
-  rescue InfluxDB::AuthenticationError => e
+  rescue InfluxDB::Error => e
     # Exception due to missing admin user
     # https://influxdb.com/docs/v0.9/administration/authentication.html
     # https://github.com/chrisduong/chef-influxdb/commit/fe730374b4164e872cbf208c06d2462c8a056a6a
     if e.to_s.include? 'create admin user'
       client.create_cluster_admin(username, password)
       updated_by_last_action true
+    else
```

Comment:
make sure to pass this through rubocop by running `rake`.  (Note: my CI is suppose to catch this but I'm scratching my head why its not running)

If you were to pass this through rubocop you will notice the preferred way is:

```ruby
rescue InfluxDB::Error => e
  raise e unless e.to_s.include? 'create admin user'

  client.create_cluster_admin(username, password)
  updated_by_last_action true
end
```